packages/data-format/index.js

```diff
@@ -0,0 +1,101 @@
+/* eslint-disable no-unused-vars */
+import Big from 'big.js'
+Big.RM = 0
+Big.NE = -19
+
+export const fromDripToCfx = numOrStr => {
+  try {
+    const cfx = new Big(numOrStr).div(1e18).toString()
+    return cfx
+  } catch (e) {
```

Comment:
Better let the user handle this error. This will make it hard to track down bugs.

Reply:
it will make the full screen white if throw a error

Reply:
done, remove all the catch, then `new Big(null)` will make the screen full white